service/src/main/kotlin/fi/espoo/evaka/daycare/service/AbsenceQueries.kt

```diff
@@ -507,7 +507,7 @@ AND daterange(bc.start_date, bc.end_date, '[]') && :period
         .groupBy({ it.first }, { it.second })
 
 data class ChildReservation(
-    val reservation: Reservation,
+    val reservation: ReservationResponse,
```

Comment:
here we already have `val createdByEvakaUserType: EvakaUserType` so we probably don't need both that and the new boolean. could one of them be removed?

Reply:
Indeed! I changed this back to using `Reservation` data type + bought back the front-end `Reservation` deserializer code etc.